pandas/tests/reshape/test_from_dummies.py

```diff
@@ -0,0 +1,313 @@
+import numpy as np
```

Comment:
Should there be a round trip test here that uses `get_dummies`?

Reply:
With round trip you mean dummify a categorical with `get_dummies` then reverse it with `from_dummies`?